pom.xml

```diff
@@ -229,7 +229,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <forkMode>always</forkMode>
+                    <forkCount>1</forkCount>
+                    <reuseForks>false</reuseForks>
```

Comment:
Unclear to me why this plugin has custom Surefire configuration to begin with, but…